samples/samples/test_autocommit.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import uuid
+
+from google.api_core.exceptions import DeadlineExceeded
+from google.cloud import spanner
+import pytest
+
+import autocommit
+
+
+def unique_instance_id():
+    """ Creates a unique id for the database. """
+    return f"test-instance-{uuid.uuid4().hex[:10]}"
```

Comment:
We should not use `f` prefix, as this feature was added in recent versions of Python. Thus, it'll probably not work on Python <=3.7

Reply:
Done